docs/source/architecture/events_and_transactions_receipts.rst

```diff
@@ -312,13 +312,13 @@ the TransactionReceiptStore after a block is validated.
 
```

Comment:
This section of Transaction Receipt Store needs an update (future item). There is now support for storing invalid transactions as well under a flag. HL Sawtooth is making use of the HL Transact library for this purpose.